docs/index.md

```diff
@@ -84,15 +84,27 @@ If you have a permissions issue then ask in [#golden-path](https://hmcts-reform.
 
 We use [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
 
-Your application will be deployed in `labs` Kubernetes namespace which has been already been created. 
-Follow the app deployment [guide](https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#application) in cnp-flux-config.
+Your application will be deployed in `labs` Kubernetes namespace which has been already been created.
 
-The cluster can be connected to with:
+For the benefit of of this tutorial we have created a separate [guide](https://github.com/hmcts/cnp-flux-config/blob/master/labs/java/README.md#creating-the-flux-config-for-your-java-lab-application) to help you create the flux config needed to deploy your lab application with flux and to enable flux to automate updating image tags to the latest version of your image.
```

Comment:
This file hasn't been pushed to master yet, here's the working link - https://github.com/hmcts/cnp-flux-config/blob/golden_path_lab_scripts/labs/java/README.md#creating-the-flux-config-for-your-java-lab-application